src/frequenz/client/common/metric/__init__.py

```diff
@@ -57,7 +57,12 @@ class Metric(Enum):
     AC_POWER_FACTOR_PHASE_2 = PBMetric.METRIC_AC_POWER_FACTOR_PHASE_2
     AC_POWER_FACTOR_PHASE_3 = PBMetric.METRIC_AC_POWER_FACTOR_PHASE_3
 
-    # AC energy metrics
+    """AC energy metrics
+    Not all components will have energy metrics as components tent to reset
+    to avoid overflow. This depends on the component model.
+    Additionally, the energy metric is inconsistent as it does not provide
+    a time parameter.
```

Comment:
typo `tent` => `tend`

Reply:
There are 3 separate points here:

- This energy metric is reported directly from the component, and not a result of aggregations i our systems. If a component does not have this metric, this field cannot be populated.
- Components that provide energy metrics reset this metric from time to time. This behaviour is specific to each component model. E.g., some components reset it on UTC 00:00:00.
- This energy metric does not specify the timestamp since when the energy was being accumulated, and therefore can be inconsistent.

In the original text, it reads like points 1 and 2 were linked. But they are not.